content/basics/vimmodes/visual.md

```diff
@@ -4,4 +4,14 @@ date =  2017-10-30T10:49:01-04:00
 weight = 7
 +++
 
-Visual Mode
+Visual Mode is where you can select words,lines,blocks,etc... There are some keys to enter in this mode, let's se them:  
+
+
+| Key          | Description   																											|
+|--------------|--------------------------------------------------------------------|
+| **v**        | This key enter will select the char under the cursor								|
+| **V**   		 | This key enter will select the line under the cursor								|
+| **ctrl+v**   | This key enter will select in vertical block under the cursor 			|
+
+------------
+> Just to remember, you can use all keys and combinations to select the text. We will see how navigate in the next step.  
```

Comment:
Just remember: you can use all navigation keys and combinations to select the text. We will see how to navigate in the next step.